CTP/common/src/com/navercorp/cubridqa/common/Constants.java

```diff
@@ -39,9 +39,18 @@
 
 	public final static String LINE_SEPARATOR = System.getProperty("line.separator");
 
-	public final static Properties COMMON_DAILYQA_CONF;
+	public static Properties COMMON_DAILYQA_CONF;
 	static {
-		COMMON_DAILYQA_CONF = CommonUtils.getConfig(CommonUtils.getEnvInFile(ENV_CTP_HOME_KEY) + File.separator + "conf" + File.separator + "dailyqa.conf");
+		try {
+			COMMON_DAILYQA_CONF = CommonUtils.getConfig(CommonUtils
+					.getEnvInFile(ENV_CTP_HOME_KEY)
+					+ File.separator
+					+ "conf"
+					+ File.separator + "common.conf");
+		} catch (Exception ex) {
+			System.out.println("[WARNING]: common.conf does not exist!");
```

Comment:
Refine message:
as is:
System.out.println("[WARNING]: common.conf does not exist!");
to be:
System.out.println("Skip common properties.");   

So, don't make customer have any concern.